### PR TITLE
Reorganized system's architecture

### LINKED ISSUE #14: Implement cookies authorization on all sites

### PR DESCRIPTION
**Pull request by Nikulin-V**
- Realized subdomains throw Flask.
- Now subdomains apps are separated packages which includes its templates and pages. Static files are still shared
- Overridden: url_for -> url (to select http/https requests)
- Now links like on the area.index page redirects to current domain (not only remote)

### CODE REVIEW

File: tools/url.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#  Nikulin Vasily © 2021
+from flask import url_for
+
+from config import SCHEME
+
+
+def url(endpoint, **kwargs):
```
Comment (markK24) [inline tools/url.py:7]: Unclear function name

safe_url_for() would be more informative

**Review by Nikulin-V — COMMENTED** [review]
Approved

**Review by PAV04 — APPROVED** [review]

File: forms/company_admin_management.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 #  Nikulin Vasily © 2021
-
```
Comment (markK24) [inline forms/company_admin_management.py:2]: What is wrong with empty lines?

**Review by markK24 — DISMISSED** [review]

File: tools/url.py (hunk 1 above)
Comment (Nikulin-V) [reply tools/url.py:7]: Name "url" is not reserved too. 
Also, this function is used in JS and HTML5 files many times, so even a few extra characters will be superfluous

File: forms/company_admin_management.py (hunk 2 above)
Comment (Nikulin-V) [reply forms/company_admin_management.py:2]: It's just a code style. Some files used to have empty line before copyrite, some didn't use to. I decided to delete them

**Review by markK24 — APPROVED** [review]

**Pull request merged**